docs/03_configuration/README.md

```diff
@@ -308,10 +309,10 @@ As mentioned earlier, the values for configuration settings defined this way ove
 is specified through any other means. So if you want to be sure that a particular configuration setting
 is defined the way you want it to be, you can use a command line option.
 
-There are various configuration settings for which it only really makes sense to use the command line option.
+There are various configuration settings for which it only makes sense to use the command line option.
 An example of this is letting the `eb` command print the EasyBuild version (via `eb --version`);
 although you could configure EasyBuild to always print its version and then exit whenever the `eb` command is
-run, that would not be very useful...
+run, that would not be less than useful...
```

Comment:
incorrect double negation now?

Reply:
Stupid English and their desire to avoid multiple negations. Of all the things they stole from germanic languages, that was a trick they never picked up.
Fixed.